### PR TITLE
[OC-90] Added Logout endpoint and removed possibility of concurrent sessions

### PR DESCRIPTION
**Pull request by FiggNewton**

### CODE REVIEW

File: Server/Services/SessionService.cs [hunk 1]
```diff
@@ -19,33 +20,33 @@ public SessionToken CreateSession(User user)
...
-            else
+
+            SessionToken newSession = new SessionToken()
             {
-                SessionToken newSession = new SessionToken()
-                {
-                    Token = EncryptionHelper.EncryptString(user.Id),
-                    User = user,
-                };
+                Token = EncryptionHelper.EncryptString(string.Concat(user.Id, DateTime.Now.ToString("MM/dd/yyyy h:mm tt"))),
```
Comment (ForceDestroy) [inline Server/Services/SessionService.cs:29]: Use DateTime.UtcNow instead

File: ServerTests/ControllerTests/LoginControllerTests.cs [hunk 2]
```diff
@@ -25,7 +25,7 @@ public LoginControllerTests()
         }
 
         [Fact]
-        public void Test_ValidateLogin_With_No_MatchingUser()
+        public void LoginController_ValidateLogin_With_No_MatchingUser()
```
Comment (ForceDestroy) [inline ServerTests/ControllerTests/LoginControllerTests.cs:28]: .....With_NoMatchingUser() to keep it consistent

File: ServerTests/ControllerTests/LoginControllerTests.cs [hunk 3]
```diff
@@ -119,7 +119,48 @@ public void Test_ValidateLogin_With_InvalidPassword()
         }
 
         [Fact]
-        public void Test_ValidateLogin_With_CorrectPassword()
+        public void LoginController_ValidateLogin_With_CorrectPassword()
+        {
+            //Initialize
+            DatabaseContext db = new DatabaseContext(_connectionString);
```
Comment (ForceDestroy) [inline ServerTests/ControllerTests/LoginControllerTests.cs:125]: Remove extra lines

**Review by ForceDestroy — CHANGES_REQUESTED** [review]
Minor Changes

**Review by Abousateur — APPROVED** [review]

**Review by ForceDestroy — APPROVED** [review]

**Pull request merged**